### PR TITLE
Add new logging errors for 4.0 releases

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst [hunk 1]
```diff
@@ -560,6 +560,54 @@ See `flight mode numbers here <https://github.com/ArduPilot/ardupilot/blob/maste
...
+.. raw:: html
+
+   </td>
+   </tr>
+
+   <tr>
+   <td>28 = Pilot Input Timeout Failsafe</td>
+   <td>
+
+- 0 = Pilot Input Failsafe Cleared
```
Comment (rmackay9) [inline common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst]: I had never heard of this pilot-input failsafe.  This really seems like an RC failsafe so I'm not sure why it exists.  It looks like it's sub only so it's really up to @jaxxzer .. I really worry this confuses non-sub users.

Comment (Hwurzburg) [reply common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst]: will add sub only to it

Comment (Hwurzburg) [reply common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst]: added

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**